node/src/accountant/db_access_objects/sent_payable_dao.rs

```diff
@@ -60,6 +61,7 @@ pub trait SentPayableDao {
     fn insert_new_records(&self, txs: &[Tx]) -> Result<(), SentPayableDaoError>;
     fn retrieve_txs(&self, condition: Option<RetrieveCondition>) -> Vec<Tx>;
     fn update_tx_blocks(&self, hash_map: &TxUpdates) -> Result<(), SentPayableDaoError>;
+    fn replace_record(&self, hash_map: &Tx) -> Result<(), SentPayableDaoError>;
```

Comment:
You say it's a hash_map here but it actually is not. Either change it to address it by a right name or the other thing, if you decide to allow supplying a collection.